india_banking/overrides/payment_order.py

```
@@ -6,29 +6,65 @@
+			self.validate_bank_balance()
 		self.validate_bank_payment_request()
 
+	def validate_bank_balance(self):
+		is_insufficient_fund = False
+		try:
+			balance = get_bank_balance(self.company_bank_account)
+
+			if flt(balance) < self.total:
+				is_insufficient_fund = True
+
+		except Exception:
```

Ruff (0.15.6)

[warning] Do not catch blind exception: `Exception`

(BLE001)